src/main/Application/Avatar/AvatarApplicationService.cs

```diff
@@ -61,13 +61,13 @@ public async Task CreateAvatarAsync()
 
                 var mappedAvatar = avatarMapperService.Apply(item, avatar);
 
-                await avatarRepository.SaveAsync(subdirectory, mappedAvatar);
+                await avatarRepository.SaveAsync(mappedAvatar);
             }
 
             var avatarServer = await avatarServerRepository.GetByAsync(configObject.Destination);
             var mappedAvatarServer = avatarServerMapperService.Apply(configObject, avatarServer);
 
-            await avatarServerRepository.SaveAsync(configObject.Destination, mappedAvatarServer!);
+            await avatarServerRepository.SaveAsync(mappedAvatarServer!);
```

Comment:
Why is there an exclamation mark beside mappedAvatarServer?

Reply:
I tried to avoid changing what isn't necessary. I believe there's a null-forgiving operator because IntelliSense flags issues when properties haven't been initialized in the constructor.